.openshift-ci/scripts/lib.sh

```diff
@@ -140,3 +140,46 @@ get_pr_details() {
     _PR_DETAILS="$pr_details"
     echo "$pr_details"
 }
+
+import_creds() {
```

Comment:
There are few examples of using mounted secrets directly, does it make sense to generalize them as well, what do you think?

Reply:
seems reasonable! I'll add in a helper or two